Library/test-helpers/lib.sh

```diff
@@ -1253,6 +1253,92 @@ limeCreateTestDir() {
 }
 
 
+true <<'=cut'
+=pod
+
+=head2 limeCopyKeylimeFile
+
+Copy file to actual dir from specified path or download file
+from repository to actual dir.
+
+    limeCopyKeylimeFile
+
+=over
+
+=back
+
+Returns 0 when the when copy or download of file to actual dir is succesfull.
+
+=cut
+
+limeCopyKeylimeFile(){
+
+    local OPTION=$1
+    local NAME=$2
+
+    if [ ${OPTION} == "-s" ] ||[ ${OPTION} == "--source" ]; then
+        # get filepath to the variblae and if file exist, copy file to actual dir
+        local FILEPATH=$(limeGetKeylimeFilePath -s ${NAME})
+        if grep -q ${NAME} <<< ${FILEPATH} ; then
+            cp ${FILEPATH} .
+        else
+            echo "File is not in installed files!"
```

Comment:
This branch is for source files so "installed files" is a bit confusing. Personally, I would use simply "File ${NAME} was not found." in both cases.

Reply:
In fact, you can use only one branch. Same parameter is passed to `limeGetKeylimeFilePath` so there is no need to have two distinct branches. Only the download one requires a separate branch.